### DISCUSSION TITLE
предложения - методы для сервиса инвентаря

### DISCUSSION THREAD

**Discussion by justqstn** [Sokol-Games-GmbH/PixelCombats.PublicApi#41] [Ideas]
ApiProp Weapon {get, set}; // id текущего оружия в каком-либо слоте. можно поменять, но если нужного оружия у игрока нету, то замены не произойдет.
Inventory.GetContext().Main.Weapon = 3; // теперь у всей комнаты оружием в основном слоте будет являться оружие с id 3. можете также выложить все id оружий

**Comment by kkohno** [comment]
как вариант можно сделать, правда указанное оружие получится выдаст не всем а только тем у кого оно куплено...
тут додумать АПИ нужно еще  - проверку куплено ли оружие тогда, чтобы можно было чтото дать тем у кого не куплено оружие

**Comment by kkohno** [comment]
или както сделать так set(1,2,3,4,5,6,7) тоесть задаем id=1 , если такое не куплено то задает 2 если и такого нет то 3 итд, если ничего нет то вообще не даст ничего
